src/main/java/lotto/Lotto.java

```diff
@@ -0,0 +1,27 @@
+package lotto;
+
+import static lotto.common.Constants.*;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Lotto {
+	private List<Integer> numbers;
```

Comment:
```suggestion
	private List<Number> numbers;
```

Integer 를 포장한 객체를 만들어보는건 어떨까요?
Number 라는 클래스 이름은 예시이고, 더 좋은 이름이 있다면 좋습니다 👍